CRM/CiviAwards/Service/AwardImport.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+use CRM_CiviAwards_Helper_CaseTypeCategory as CaseTypeCategory;
+
+/**
+ * Utility class for importing Awards using CSV importer extension.
+ */
+class CRM_CiviAwards_Service_AwardImport {
+
+  /**
+   * Create an Award and the related entities.
+   *
+   * @param array $params
+   *   Information for creating the award and related entities.
+   */
+  public function create(array $params) {
+    $tx = new CRM_Core_Transaction();
+
+    $params['case_type_category'] = $this->getCaseCategoryValueForAwards();
+    $params['definition'] = $this->createDefinition($params);
```

Comment:
Yep. We should keep this, this is a good one as the previous import was not importing the definitions successfully. I think we should just keep the `activity_types` and `caseRoles` defaults to create this column so that it can be populated from start and remove other inputs.

Reply:
Ok, agree with that. I left only the default values here, and it the same result than creating the Award using the interface. Done here: https://github.com/compucorp/uk.co.compucorp.civiawards/pull/164/commits/e42e816414f9f5801468d3505ea7a1afc599db71